tools/sggolangcilintv2/tools.go

```diff
@@ -119,6 +125,57 @@ func Fix(ctx context.Context, config Config, args ...string) error {
 	return nil
 }
 
+// Run `golangci-lint fmt --diff` in every Go module from the root of the current git repo.
+func Fmt(ctx context.Context, config Config, args ...string) error {
```

Comment:
We previously just provided a target that always wrote files like `FmtFix`. As we typically run git to verify no diff in the end, I think it's find to let `Fmt` write (i.e. be `FmtFix`). No strong opinion though, but less code to maintain 🤓

Reply:
Right, makes sense. You can also just pass the `--diff` argument if you don't want it to write any files. 👍

Reply:
Fixed!